packages/cli/lib/moduleInterfaces/ApplicationInterface.ts

```diff
@@ -16,25 +16,18 @@
  * limitations under the License.
  */
 
```

Comment:
All the other files don't have a space here

Reply:
Fixed.